pkg/e2e/osd/dedicatedadmin.go

```diff
@@ -269,6 +271,27 @@ var _ = ginkgo.Describe(dedicatedAdminTestName, func() {
 			err := manageSubscriptions(ctx, namespaceList, h)
 			Expect(err).NotTo(HaveOccurred())
 		}, float64(viper.GetFloat64(config.Tests.PollingTimeout)))
+
+		util.GinkgoIt("dedicated-admin user can patch consoles.operator.openshift.io CR", func(ctx context.Context) {
+			// Impersonate ded-admin
+			h.Impersonate(rest.ImpersonationConfig{
+				UserName: "dummy-admin@redhat.com",
+				Groups: []string{
+					"dedicated-admin",
```

Comment:
The group name needs to be corrected here for the test to pass

```suggestion
					"dedicated-admins",
```

Reply:
Thanks @mrbarge  for the review. I have updated this in my latest commit.